pkg/descheduler/strategies/failedpods.go

```diff
@@ -17,6 +17,8 @@ import (
 	"sigs.k8s.io/descheduler/pkg/descheduler/strategies/validation"
 )
 
+const FailedPodsStrategyName = "FailedPods"
```

Comment:
```suggestion
const FailedPodsStrategyName = "FailedPod"
```
My current monitors/metrics show `failedpod`. Looking at the last [release](https://github.com/kubernetes-sigs/descheduler/blob/v0.24.0/pkg/descheduler/strategies/failedpods.go#L83), it also shows `FailedPod`

Reply:
thanks for catching, fixed